app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
```

Comment:
yeah, i tend to not necessarily make that particular assumption (or at least discovering it very early in the rails console).  I'm not arguing against only making models hold AR classes, just that a) there is no global convention and b) we haven't set a convention for our project.   So i think if we want to make some convention we should document it and fix any existing classes.